.github/workflows/main.yml

```diff
@@ -98,18 +98,23 @@ jobs:
         with:
           name: control-dist
           path: ./**
-  build_binary:
-    name: 'Build'
+  GoReleaser:
+    name: 'GoReleaser'
     needs:
       - build_control
       - Test_sqlite
       - Test_postgres
+      - Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/setup-go@v2
```

Comment:
Several Github actions are not up to date, we could maybe use Dependabot?
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot